index/server/.dockerignore

```diff
@@ -0,0 +1,11 @@
+#registry-viewer dockerignore fields
+registry-viewer/tests
+registry-viewer/.vscode
+registry-viewer/.github
+registry-viewer/cypress
+registry-viewer/cypress.json
+registry-viewer/jest.config.js
+registry-viewer/jest.setup.js
+registry-viewer/LICENSE
+registry-viewer/OWNERSHIP
```

Comment:
should be OWNERS, and no cypress

I wonder if there is a better way to handle this

Reply:
This is just copied from https://github.com/devfile/registry-viewer/blob/main/.dockerignore

TBH, I should probably just updated the build.sh script to copy this file over from registry-viewer, rather than maintaining a separate copy